src/renderer/components/ExportLogsButton.js

```diff
@@ -105,4 +111,9 @@ const ExportLogsBtn = ({ hookToShortcut, primary = true, small = true, title, ..
   );
 };
 
-export default ExportLogsBtn;
+// $FlowFixMe (?)
+export default connect(
+  createStructuredSelector({
+    allAccounts: accountsSelector,
+  }),
+)(ExportLogsBtn);
```

Comment:
small detail, but can you prefer using the `useSelector` hook?

Reply:
indeed thats better